src/nuclide_header.F90

```diff
@@ -539,6 +592,18 @@ subroutine nuclide_create_derived(this)
           this % sum_xs(t) % total(j:j+n-1) = this % sum_xs(t) % total(j:j+n-1) + &
                rx % xs(t) % value
 
+          ! Calculate nu-photon total cross section
```

Comment:
We'll need to be careful because the photon production data is a bit messy for some evaluations. Often, you'll see photon production assigned to summed reactions like MT=4 or MT=3. Also, sometimes photon production is listed for reactions that don't have any corresponding cross sections. If you look at how we handle this in `openmc.data` you'll see that we have a `summed_reactions` dictionary that stores redundant reactions, some of which have photon production that doesn't appear elsewhere. We'll need to make sure that the total photon production that is listed in the ACE file is actually what we end up using. Anyway, for now you don't need to start digging into this too deeply but this is something we'll have to keep in mind as a TODO.